src/apps/activities/domain/response_values.py

```diff
@@ -329,6 +337,34 @@ class AudioPlayerValues(PublicModel):
     file: str | None = Field(default=None)
 
 
+class _PhrasalTemplateTextField(PublicModel):
+    type: PhrasalTemplateFieldType = PhrasalTemplateFieldType.SENTENCE
+    text: str
+
+
+class _PhrasalTemplateItemResponseField(PublicModel):
+    type: PhrasalTemplateFieldType = PhrasalTemplateFieldType.ITEM_RESPONSE
+    id: str | None = None
```

Comment:
I think this field should be required, and of type `uuid.UUID`
```suggestion
    id: uuid.UUID
```

Also, I think we should have some logic that restricts this id in the following ways:
- It can only reference items in the same activity as the phrasal template
- The referenced item should be one of the supported item types (`text`, `singleSelect`, `multiSelect`, `slider`, `numberSelect`, `timeRange`, `date`, `sliderRows`, `singleSelectRows`, `multiSelectRows`)

Reply:
it can't be a UUID, because it stored as json in database and UUID is not JSON serializable

`sqlalchemy.exc.StatementError: (builtins.TypeError) Object of type UUID is not JSON serializable`

Reply:
I notice that in other item types the solution was add a validation
```    @validator("id")
    def validate_id(cls, value):
        return validate_uuid(value)```
        
so I will add it

But, we can't validate if this item id is in referenced activity and also can't validate its type, but in context of item creating we don't have the activity, all validation in done during item object creating

To do this kind of validation we need to change the way we create item types

Reply:
That's a good point. It also seems that the way these items are created is through the Applet Create (`/workspaces/{owner_id}/applets`) endpoint. Is that correct? Is yes, then at that point there are no item IDs to reference so I wonder where these `id` values will come from.

Reply:
change it to item_name and add validations